### PR TITLE
Homework 7 completed

### PR DESCRIPTION
**Pull request by AleksandrDark**

### CODE REVIEW

**Review by troe88 — CHANGES_REQUESTED** [review]
This MR should consists of files, that have realtion to the HW-7 only.
Delete the rest of it from MR, please.

**Comment by AleksandrDark** [comment]
> This MR should consists of files, that have realtion to the HW-7 only. Delete the rest of it from MR, please.

File: src/test/java/com/epam/tc/hw7/util/JdiDataProvider.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.Map;
+import org.testng.annotations.DataProvider;
+
+public class JdiDataProvider {
+    public static final String EXPECTED_USERNAME = "ROMAN IOVLEV";
```
Comment (troe88) [inline src/test/java/com/epam/tc/hw7/util/JdiDataProvider.java]: What is the reason of separating "full name" and name&password from User entity??

**Review by troe88 — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/tc/hw7/util/JdiDataProvider.java (hunk 1 above)
Comment (AleksandrDark) [reply src/test/java/com/epam/tc/hw7/util/JdiDataProvider.java]: no particular reason, it just seems logical to me. 
maybe should be combined with a user class

**Pull request merged**